app/views/campaigns/show.html.haml

```diff
@@ -62,10 +62,11 @@
         .mui-row
           .mui-col-md-12
             .share-campaign
-              %label Share:
-              %ul
+              %label= "Share:"
+              %ul.artist-social-share
                 %li
-                  = social_share_button_tag(@campaign.title, allow_sites: %w(twitter facebook))
+                  %a{href: "http://twitter.com/home?status=#{campaign_path}", target: "_blank ", title: "Share on Twitter", image: @campaign.image}= icon('fab', 'twitter')
```

Comment:
Is the `social_share_button_tag` not working for us?  Does this tag come from a gem? I'm totally okay with not using the gem and we can use your solution for now. I would like to extract this to a helper method and/or use rails helpers. But this solution is totally okay for now.